FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -658,14 +652,12 @@ - (void)requestAlwaysAuthorization {
     }
```

Comment:
the above `respondsToSelector:` calls are now unnecessary

Reply:
ahh good catch, I will fix this